lib/extras/codec_jpg.cc

```diff
@@ -503,7 +498,7 @@ Status EncodeImageJPG(const CodecInOut* io, JpegEncoder encoder, size_t quality,
       break;
     case JpegEncoder::kSJpeg:
       JXL_RETURN_IF_ERROR(
-          EncodeWithSJpeg(ib, quality, chroma_subsampling, bytes));
+          EncodeWithSJpeg(ib, io, quality, chroma_subsampling, bytes));
```

Comment:
could we pass just the blobs here?

Reply:
This is symmetric with the libjpeg function. I have no strong opinion on this though.